code/obj/flock/cage.dm

```diff
@@ -164,8 +164,13 @@
 	process()
 		// process fluids into stuff
 		if(reagents.has_reagent(target_fluid, create_egg_at_fluid))
+			if (src.flock?.getComplexDroneCount() < FLOCK_DRONE_LIMIT)
+				spawnEgg()
+			else
+				var/obj/item/flockcache/cube = new(get_turf(src))
+				cube.resources = create_egg_at_fluid
```

Comment:
I think this could result in several hundred resources produced per person caged?

Reply:
Yep, but it's the same ratio as would have been converted into drones so I think it's okay? Also bear in mind this only happens when flock is already massive and needs to be building structures anyway.